jingo/__init__.py

```diff
@@ -9,12 +9,17 @@
 
 from django.conf import settings
 from django.template.base import Origin, TemplateDoesNotExist
-from django.template.context import get_standard_processors
 from django.template.loader import BaseLoader
 from django.utils.importlib import import_module
 
 import jinja2
 
+try:
+    from django.template.engine import Engine
```

Comment:
can we just put the `get_standard_processors` definition here, so we don't need to do the `Engine = None` trick or worry about `import _ as ...`? I.e. something like this, except PEP8ed:

```python
try:
    from django.template.engine import Engine
    def get_standard_processors():
        return Engine.get_default().template_context_processors
except ImportError:
    from django.template.context import get_standard_processors
```